frontend/src/App.tsx

```diff
@@ -4,6 +4,9 @@ import GlobalStyles from '@mui/material/GlobalStyles';
 import HomeScreen from 'screens/HomeScreen';
 import NavBar from 'components/Header/NavBar';
 import Footer from 'components/Footer/Footer';
+import LoginPage from 'components/LoginPage/LoginPage';
+import ForgotPassword from 'components/ForgotPassword/ForgotPassword';
+import Registration from './components/Registration/Registration';
```

Comment:
You called login as "LoginPage", but registration and forgot password without "page" prefix. Choose one variant how to call imported variables.

Reply:
done